src/components/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const defaultColor = 'rgb(249, 144, 77)'
+function Button (props) {
+  const style = {
+    width: props.wide ? '50%' : '25%',
+    background: props.color ? props.color : defaultColor
+  }
+  return (
+    <button style={style} onClick={() => props.clickHandler(props.name)}>{props.name}</button>
+  )
+}
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+  color: PropTypes.string.isRequired,
```

Comment:
`wide` and `color` props shouldn't be `isRequired` because there are some `Button` renders in `ButtonPanel` to which we aren't passing either of those two `props`. We should just provide these with defaultProps (and remove `isRequired` from them). OR in `ButtonPanel` we should pass ALL `Button` renders with `color` and `wide` props.

![image](https://user-images.githubusercontent.com/6015315/61988745-32f46980-b058-11e9-90fe-1f9b3a7ae1c1.png)

```javascript
Button.defaultProps = {
  color: defaultColor,
  wide: false
}
```